manim/animation/indication.py

```diff
@@ -342,7 +343,23 @@ def __init__(self, vmobject, n_segments=10, time_width=0.1, remover=True, **kwar
 #   Remove?
 #   Deprecate?
 #   Keep and add docs?
+@deprecated(reason="Use Create then FadeOut to achieve this effect.")
 class ShowCreationThenFadeOut(Succession):
+    """Creates mobject first, then fades out.
```

Comment:
Fixed a typo and made it so that the animations are linked. See https://docs.manim.community/en/stable/contributing/references.html for more details.
```suggestion
    """Applies :class:`~.Create`s on a mobject, then :class:`~.FadeOut`.
```